client/src/components/OrgBook/OrgBookTableOfContents.js

```diff
@@ -0,0 +1,290 @@
+import React, { useState, useCallback, useEffect } from "react";
+import styled from "styled-components";
+import { theme, mq } from "../../constants/theme";
+import { useTranslation } from "react-i18next";
+import { WhiteSpace } from "antd-mobile";
+import { Avatar } from "components/Avatar";
+import BackArrowButton from "./OrgBookBackButton";
+import { getInitialsFromFullName } from "utils/userInfo";
+import GTM from "../../constants/gtm-tags";
+import SvgIcon from "../Icon/SvgIcon";
+import draftPageIcon from "../../assets/icons/orgbook-draft-page.svg";
+import livePageIcon from "../../assets/icons/orgbook-live-page.svg";
+import lockClosedIcon from "../../assets/icons/orgbook-lock-closed.svg";
+import lockOpenIcon from "../../assets/icons/orgbook-lock-open.svg";
+import pageIcon from "../../assets/icons/orgbook-page.svg";
+import plusIcon from "../../assets/icons/orgbook-plus.svg";
+import { OrgBookStyledModalContainer } from "./OrgBookStyledModal";
+
+const { colors, typography } = theme;
+const { white, black, lightGray, mediumGray } = colors;
+
+const OrgBookTOCHeader = styled.div`
+  background-color: rgb(120, 120, 120);
+  color: ${white};
+  height: 3vh;
+  padding: 1.4rem 1.4rem 3.4rem 1.4rem;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+`;
+const OrgBookTOCWrapper = styled.div`
+  opacity: ${(props) => (props.selectedDirty ? "0.5" : "1")};
+`;
+
+const OrgBookHeaderLabelContainer = styled.div``;
+
+const HeaderSpacerContainer = styled.div`
+  width: 50px;
+`;
+
+const MainNavigationContainer = styled.div`
+  color: ${white};
+  overflow-y: auto;
+  scroll-behavior: smooth;
+  scrollbar-color: light;
+  padding: 1.4rem 1.4rem 3.4rem 1.4rem;
+`;
+
+const OrgAvatarAndNameContainer = styled.div`
+  display: inline-block;
+  height: 2vh;
+  padding: 2.4rem 1.4rem 3.4rem 2.4rem;
+`;
+
+const PageListContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  flex-wrap: nowrap;
+  height: 4vh;
+  margin: 1.4rem 1.4rem 1.4rem 6.4rem;
+  cursor: pointer;
+  :hover {
+    background-color: ${lightGray};
+    color: ${black};
+  }
+  border: ${(props) => (props.selected ? "1px solid white" : "none")};
+  pointer-events: ${(props) => (props.selectedDirty ? "none" : "auto")};
+`;
+
+const AddNewPageContainer = styled.div`
```

Comment:
Looks like lot of similar css properties for both styled components? DO you need it at both places, how about creating one and using same with updated properties for other one.

Reply:
I hesitate to make styling structure changes at this point, unless they're bugs.   But yes, there are probably duplicate properties.   We are under a very hard deadline here, and it's not worth taking the chance on breaking things at this point.   Thanks for the catch.